README.md

```diff
@@ -1,40 +1,56 @@
 ![logo](https://upload.wikimedia.org/wikipedia/commons/b/b5/Kore_2g_logo.png)
 
-* OpenKore is an custom client and intelligent automated assistant for Ragnarok Online.
-* It is a **free**, open source and cross-platform program (Linux, Windows and MacOS are supported).
+* OpenKore is an custom client & intelligent automated assistant for Ragnarok Online.
+* It is a **free**, open source & cross-platform program _(Linux, Windows & MacOS are supported)_.
 
-## Prerequisites
+## Prerequisites.
 
 To run OpenKore you will need:
 * [Read the Requirements page on our wiki](http://wiki.openkore.com/index.php/How_to_run_OpenKore#Requirements)
 
-## Quickstart
+## Quickstart.
 
-1. [Download OpenKore](https://github.com/OpenKore/openkore/archive/master.zip) and extract it. Alternatively, you could press the **Windows Key + R**, type in ``cmd`` and enter. Run the following command in the cmd to clone.
+1. [Download OpenKore](https://github.com/OpenKore/openkore/archive/master.zip) & extract it. Alternatively, you could press the **Windows Key + R**, type in ``cmd`` & enter. Run the following command in the cmd to clone.
 ***Note: [Git](https://git-scm.com/) required.***
 ```
 git clone https://github.com/OpenKore/openkore.git
 ```
 
 2. Configure OpenKore: [documentation](http://openkore.com/index.php/Category:Control).
-3. Run openkore.pl (you can run start.exe or wxstart.exe if you use Windows).
-
-## Things to know
-
-* Before submitting an issue, ensure you are running the latest commit on branch.
-* Check if an issue already exists for your request.
+3. Run openkore.pl _(You can run start.exe or wxstart.exe if you use Windows)_.
+
+## F.A.Q. (Frequently Asked Questions).
+<!-- Source: http://forums.openkore.com/viewtopic.php?f=0&t=11287 -->
+ 1. **Have a problem?**
+    - Update your openkore.
+    - Download anew.
```

Comment:
new .

Reply:
It's literally mean the same but with proper English.

Reply:
i though that it was wrong too hahahah
i had to reasearch to know that is correct

Reply:
I think use `a new` ?